### DISCUSSION TITLE
How to change the Topic name of each HSR when using two real HSRs

### DISCUSSION THREAD

**Discussion by Shoichi-Hasegawa0628** [HumanSupportRobot/QandA#100] [Q&A]
I am currently trying to do a research using two HSRs.
I think there is a problem that the topic name of each HSR is covered by each other.
In that case, how can I change the topic name?

**Comment by TomohiroOno33** [comment]
Is your problem the same as [this page](https://github.com/orgs/HumanSupportRobot/discussions/68)?
